db/db_test.cc

```diff
@@ -2585,10 +2585,10 @@ class ModelDB : public DB {
     return Status::NotSupported("Not implemented.");
   }
 
-  Status VerifyChecksum() override {
```

Comment:
should keep this version as well?

Reply:
The default implementation of DB::VerifyChecksum() will call the other implementation. Also, this is unit test code. If the test code doesn't use the function, I don't think we should worry about it.

Reply:
Interestingly, Windows complains about it. Let me add it back.

Reply:
Not Windows, Travis.